### PR TITLE
Fix broken links in the guides

### PR DESCRIPTION
**Pull request by michelle-rago**
## Changes proposed in this pull request:

- These broken links were noted in a Siteimprove report.
- A few of the commits have a comment where some explanation might be helpful.

### CODE REVIEW

**Review by quarterback — APPROVED** [review]

**Review by mgwalker — APPROVED** [review]
Giving this the required "owners" approval, since Ron approved it and that's good enough for me.

File: content/engineering/about/resources.md [hunk 1]
```diff
@@ -30,4 +30,4 @@ This guide evolved from work spearheaded by the 18F Engineering Chapter and the
 - [Cloud.gov Production-ready guide](https://cloud.gov/docs/deployment/production-ready/)
 - [Login.gov Identity Playbook](https://login.gov/playbook/)
 - [Digital.gov Guides and Resources](https://digital.gov/resources/)
-- [GSA Tech Guides](https://tech.gsa.gov/guides/opensource/)
+- [GSA Tech Guides](https://tech.gsa.gov/)
```
Comment (mgwalker) [inline content/engineering/about/resources.md:33]:
```suggestion
- [GSA Tech Guides](https://tech.gsa.gov/guides)
```

File: content/engineering/tools/accessibility-scanning.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ Install pa11y locally:
 
 Create a `.pa11yci` in the root of your directory to [configure your pa11y CI run](https://github.com/pa11y/pa11y-ci#configuration). This step is optional, but helpful if you want to change any default configuration.
 
-For an example, take a look at the [18F guides site's .pallyci file](https://github.com/18F/guides/blob/main/.pa11yci).
+For an example, take a look at the [18F guides site's .pallyci file](https://github.com/18F/guides/blob/main/.pa11yci.js).
```
Comment (mgwalker) [inline content/engineering/tools/accessibility-scanning.md:62]:
```suggestion
For an example, take a look at the [18F guides site's .pa11yc.jsi file](https://github.com/18F/guides/blob/main/.pa11yci.js).
```

**Review by mgwalker — COMMENTED** [review]
Adding two small changes here. I'll merge them and then approve.

**Review by mgwalker — APPROVED** [review]

**Comment by michelle-rago** [comment]
Thank you, @mgwalker!

**Review by durland18F — APPROVED** [review]

**Review by jasnakai — APPROVED** [review]
LGTM!

**Comment by michelle-rago** [comment]
How many approvals do I need???

**Comment by beechnut** [comment]
> Waiting on code owner review from 18F/18f-engineering-leadership and/or quarterback.

So, either someone on the @18F/18f-engineering-leadership group or @quarterback have to approve the latest set of changes.

**Comment by mgwalker** [comment]
Bah! I had previously approved them, but I guess it revokes approvals when there are changes now. Approval incoming!

**Review by mgwalker — APPROVED** [review]
Approval number three!

**Comment by mgwalker** [comment]
This flow is like if Concur built approval. Just keep approving! Eventually it'll go!

**Comment by michelle-rago** [comment]
Guess we still need the quarterback...this is fun! 

Next time I won't mash up changes from different guides into one PR (I guess that's better?)

**Review by quarterback — APPROVED** [review]

**Pull request merged**